api-client/src/runs/addCameraImageSettingsToRun.ts

```diff
@@ -0,0 +1,18 @@
+import { POST, request } from '../request'
+
+import type { CameraResponse, CameraSettings } from '../camera'
+import type { ResponsePromise } from '../request'
+import type { HostConfig } from '../types'
+
+export function addCameraImageSettingsToRun(
+  config: HostConfig,
+  runId: string,
+  data: CameraSettings
+): ResponsePromise<CameraResponse> {
+  return request<CameraResponse, { data: CameraSettings }>(
```

Comment:
```suggestion
  return request<CameraSettingsResponse, { data: CameraSettings }>(
```
See comment elsewhere